packages/pouchdb-replication/src/index.js

```diff
@@ -4,6 +4,24 @@ import sync from './sync';
 function replication(PouchDB) {
   PouchDB.replicate = replicate;
   PouchDB.sync = sync;
+
+  Object.defineProperty(PouchDB.prototype, 'replicate', {
```

Comment:
Since this is `object.foo.bar` instead of `object.bar`, I don't believe we can access the `this` any other way. I believe `replicate` has to be a dynamic computed property that grabs the `this` and then binds it to its children (`to` and `from`). If there's another way, though, then I'm all for that.